src/irmin-pack/checks.ml

```diff
@@ -156,6 +156,26 @@ struct
       Cmdliner.Term.(const (fun root () -> Lwt_main.run (run ~root)) $ path)
   end
 
+  module Rebuild_index = struct
```

Comment:
It's a detail, but can you use the same name, either `rebuild` or `reconstruct`? Either one works for me, I have a slight preference for `reconstruct_index` because there is already a `reconstruct` command for storage in tezos and to be consistent with that.

Reply:
No problem! I just pushed a commit renaming the command from `rebuild-index` to `reconstruct-index` and fixes the names of the functions/module you pointed out too.